notNeededPackages.json

```diff
@@ -210,6 +210,12 @@
             "sourceRepoURL": "https://github.com/blakeembrey/camel-case",
             "asOfVersion": "1.2.1"
         },
+        {
+            "libraryName": "catalog",
```

Comment:
According to the docs this should be the descriptive name, i.e. “Catalog” 🤷‍♀️ To clarify what this package is we could call it “Catalog Living Style Guide” to help readers of notNeededPackages.json understand what it is …